### ISSUE TITLE
Notify that e2ee isn't supported when joining an e2ee room

### ISSUE THREAD

**Issue by HarHarLinks** [matrix-org/matrix-hookshot#342]
**Is your feature request related to a problem? Please describe.**
This is a hard to spot error, but hookshot can still be useful in e2ee rooms even without crypto support: once set up, post its messages unencrypted.

**Describe the solution you'd like**
Notify that e2ee isn't supported when joining an e2ee room

**Describe alternatives you've considered**
Replying to each message it can't decrypt isn't too great.

**Additional context**
Auto-creating the widget, once it supports all services, would be a great workaround. In my case it didn't do that and I needed to send the widget creation command to the room manually from an unencrypted client.

**Comment by Half-Shot** [comment]
> Auto-creating the widget, once it supports all services, would be a great workaround. In my case it didn't do that and I needed to send the widget creation command to the room manually from an unencrypted client.

I think this is probably the best approach.

**Comment by HarHarLinks** [comment]
Yeah that should probably be done in most cases upon join unless disabled (can it also be removed automatically when kicking the bot out of the room?).
But I think as long as e2ee isn't supported, it should still warn users that its ~cli~ chat command interface is practically disabled.

**Comment by Half-Shot** [comment]
I would actually say the one thing we should do is make the widget creation only happen when the user asks for it (by clicking on a reaction perhaps? though that's basically exploiting a weakness in Matrix's encryption system..).
